vault/logical_system_quotas.go

```diff
@@ -212,10 +232,11 @@ func (b *SystemBackend) handleRateLimitQuotasRead() framework.OperationFunc {
 		}
 
 		data := map[string]interface{}{
-			"type": qType,
-			"name": rlq.Name,
-			"path": nsPath + rlq.MountPath,
-			"rate": rlq.Rate,
+			"type":     qType,
+			"name":     rlq.Name,
+			"path":     nsPath + rlq.MountPath,
+			"rate":     rlq.Rate,
+			"interval": rlq.Interval.String(),
```

Comment:
We may not want to return `<number>s` format over the API. How about returning just the integer?

Reply:
e.g. `100000` vs `1s`, the later is cleaner to me. Should the output not also be reflective of the input?

Reply:
We could return in the unit of second (just `1`) instead of `10000`.

`TypeDurationSecond` accepts both `1s` and `1`. The rationale behind the comment is that with input, we have control over how we parse it, and we do support parsing of both Go style durations and plain integers. Now assume that the input was just `1` then the output with the PR as it stands would still be `1s` which is inconsistent with the input.

Historically we have moved away from returning Go style durations because it puts more onus on output consuming non-go systems to parse it.

Reply:
> Now assume that the input was just 1 then the output with the PR as it stands would still be 1s which is inconsistent with the input.

If the user provides `1`, I would imagine this really means `1s`, in which case the output of `1s` would also make sense. Is this not the case? If not, how is the value `1` interpreted?

Reply:
Yep